src/test/runTest.ts

```diff
@@ -13,7 +14,11 @@ async function main() {
 		const extensionTestsPath = path.resolve(__dirname, './suite/index');
 
 		// Download VS Code, unzip it and run the integration test
-		await runTests({ extensionDevelopmentPath, extensionTestsPath });
+		await runTests({
+			extensionDevelopmentPath,
+			extensionTestsPath,
+			launchArgs: ['--user-data-dir', `${os.tmpdir()}`],
```

Comment:
Fixes issue on macOS when tests run twice in a row and throw error: `Could not delete obsolete instance`.